exchanges/gateio/gateio_types.go

```diff
@@ -485,17 +485,25 @@ type CurrencyInfo struct {
 
 // CurrencyPairDetail represents a single currency pair detail.
 type CurrencyPairDetail struct {
-	ID              string       `json:"id"`
-	Base            string       `json:"base"`
-	Quote           string       `json:"quote"`
-	TradingFee      types.Number `json:"fee"`
-	MinBaseAmount   types.Number `json:"min_base_amount"`
-	MinQuoteAmount  types.Number `json:"min_quote_amount"`
-	AmountPrecision float64      `json:"amount_precision"` // Amount scale
-	Precision       float64      `json:"precision"`        // Price scale
-	TradeStatus     string       `json:"trade_status"`
-	SellStart       float64      `json:"sell_start"`
-	BuyStart        float64      `json:"buy_start"`
+	ID              currency.Pair `json:"id"`
+	Base            currency.Code `json:"base"`
+	BaseName        string        `json:"base_name"`
+	Quote           currency.Code `json:"quote"`
+	QuoteName       string        `json:"quote_name"`
+	Fee             types.Number  `json:"fee"`
+	MinBaseAmount   types.Number  `json:"min_base_amount"`
+	MinQuoteAmount  types.Number  `json:"min_quote_amount"`
+	MaxBaseAmount   types.Number  `json:"max_base_amount"`
+	MaxQuoteAmount  types.Number  `json:"max_quote_amount"`
+	AmountPrecision float64       `json:"amount_precision"` // Amount scale
+	Precision       float64       `json:"precision"`        // Price scale
```

Comment:
🐒  Still don't subscribe to the mindset
> _the sins of the ~father~ exchange shalt be visited on all the developers_
>  (Knuth - Probably 🤷)
Where it makes sense to sanitise the crazy, lets be handing out the Clozaril freely 💊 
🌴 1️⃣ 
```suggestion
	AmountPrecision float64       `json:"amount_precision"`
	PricePrecision       float64       `json:"precision"`
```